iron-ajax.html

```diff
@@ -242,7 +242,7 @@
         notify: true,
         readOnly: true,
         value: function() {
-          this._setActiveRequests([]);
```

Comment:
Fun fact - the compiler pass is now also verifying the return type of default value functions match the annotated type of the property :-).